app/views/devise/registrations/edit.html.erb

```diff
@@ -13,6 +13,11 @@
     <%= f.text_field :username %>
   </div>
 
+  <div class="field">
+   <%= f.label :default_location %><br />
```

Comment:
Maybe add something to explain what this is?

Like `<%= f.label 'Default Location for screenings (where you watched the movie)' %><br />`